helm/fuel-block-committer/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: fuel-canary-watchtower
+description: A Helm chart for Kubernetes
+type: application
+version: 0.5.1
```

```suggestion
version: 0.1.0
```

Was this version just copied from the block committer?